app/scripts/modules/core/src/presentation/forms/inputs/ReactSelectInput.tsx

```diff
@@ -92,6 +92,8 @@ export function ReactSelectInput<T = string>(props: IReactSelectInputProps<T>) {
       <TetheredSelect {...commonProps} options={options} />
     ) : mode === 'VIRTUALIZED' ? (
       <VirtualizedSelect {...commonProps} options={options} optionRenderer={null} />
+    ) : mode === 'CREATABLE' ? (
+      <CreatableSelect {...commonProps} options={options} />
```

Comment:
So the `CreatableSelect` is neither tethered nor virtualized, right? I wonder if that should be called out anywhere. 🤷‍♂️

Reply:
correct it is neither, like the base SelectInput.  I'm wondering: did you have an expectation that it might be tethered or virtualized?